yellowbrick/text/tsne.py

```diff
@@ -35,7 +35,7 @@
 ##########################################################################
 
 def tsne(X, y=None, ax=None, decompose='svd', decompose_by=50, classes=None,
-           colors=None, colormap=None, **kwargs):
+           colors=None, colormap=None, alpha=1.0, **kwargs):
```

Comment:
If possible, I'd also like to maintain the default `alpha=0.7` that we had before as it makes it easier to identify clusters with lots of points that are very close together in the TSNE projection.

Reply:
Good point! Done!